src/api/permissionsTypes.js

```diff
@@ -0,0 +1,75 @@
+/* @flow strict-local */
+
+import { typesEquivalent } from '../generics';
+import { objectValues } from '../flowPonyfill';
+
+/**
+ * Organization-level role of a user.
+ */
+// Ideally both the type and enum would have the simple name; but a type
+// and value sharing a name is one nice TS feature that Flow lacks.
+// Or we could leapfrog TS and make this a Flow enum:
+//   https://flow.org/en/docs/enums/
+// (TS has its own enums, but they are a mess.)
+// eslint-disable-next-line flowtype/type-id-match
+export type RoleT = 100 | 200 | 300 | 400 | 600;
+
+/**
+ * An enum of all valid values for `RoleT`.
+ *
+ * See RoleValues for the list of values.
+ */
+export const Role = {
+  Owner: (100: 100),
+  Admin: (200: 200),
+  Moderator: (300: 300),
+  Member: (400: 400),
+  Guest: (600: 600),
```

Comment:
Let's include in this jsdoc a link to somewhere in the API docs that describes these values. (I think there are several places it appears; any will do.)

Quite possibly #5382 should do that too, but it's less essential there because the type definition is right next to a place where it's used, which has such a link.